docs/source/ml.rst

```diff
@@ -39,23 +39,40 @@ values in a column. The "sparse_*" models include a LASSO regression step
-sparse_ridge_regressor         regression          `Ridge <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html>`_ 
-gradient_boosting_regressor    regression          `GradientBoostingRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html>`_      ``n_estimators=500, max_depth=2``
-random_forest_regressor        regression          `RandomForestRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html>`_              ``n_estimators=500``
-extra_trees_regressor          regression          `ExtraTreesRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html>`_                  ``n_estimators=500``
-=============================  ================    ==================================================================================================================================   ==================================
-
+as indicated in the "Altered Defaults" column. All models also have
+``random_state=42``. Note that "multilayer_perceptron_classifier"
+and "multilayer_perceptron_regressor" can only be used with
+hyperband; see :ref:`hyperparam-search`.
+
+================================  ================    ==================================================================================================================================   ==================================
+Name                              Model Type          Algorithm                                                                                                                            Altered Defaults
+================================  ================    ==================================================================================================================================   ==================================
+sparse_logistic                   classification      `LogisticRegression <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html>`_                ``C=499999950, tol=1e-08``
+gradient_boosting_classifier      classification      `GradientBoostingClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html>`_    ``n_estimators=500, max_depth=2``
+random_forest_classifier          classification      `RandomForestClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html>`_            ``n_estimators=500``
+extra_trees_classifier            classification      `ExtraTreesClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html>`_                ``n_estimators=500``
+multilayer_perceptron_classifier  classification      `MLPClassifier <https://github.com/civisanalytics/muffnn>`_ 
+stacking_classifier               classification      `StackedClassifier <GET_LINK_FROM_CIVISMLEXT>`_ 
```

Comment:
What is this `GET_LINK_FROM_CIVISMLEXT` do?

Reply:
Ooops. This was a placeholder that I didn't catch. When I started writing these docs, the open sourcing was still in progress, so I didn't have anything to link to. Good catch!